plugins/BEdita/Core/src/Model/Table/LocationsTable.php

```diff
@@ -51,24 +51,20 @@ public function initialize(array $config)
 
         $this->extensionOf('Objects');
 
-        $this->addBehavior('BEdita/Core.Relations');
-
-        $this->addBehavior('BEdita/Core.Searchable', [
-            'fields' => [
-                'title' => 10,
-                'description' => 7,
-                'body' => 5,
-                'address' => 1,
-                'locality' => 2,
-                'country_name' => 2,
-                'region' => 2,
-            ],
-        ]);
+        if ($this->hasBehavior('Searchable')) {
```

Comment:
Maybe not necessary since its parent adds `Searchable` behavior. Isn't it?
In case we could remove the check on behaviors added by `ObjectsBaseTable` in other tables too.

Reply:
Right, check is not necessary. If for some reasons these behaviors are remove we would have other bad side effects.